src/engine/shared/console.cpp

```diff
@@ -109,6 +109,46 @@ const IConsole::CCommandInfo *CConsole::FirstCommandInfo(int AccessLevel, int Fl
 	return nullptr;
 }
 
+int CConsole::AccessLevelToInt(const char *pAccessLevel)
+{
+	// alias for old legacy integer access levels
+	if(!str_comp_nocase(pAccessLevel, "0"))
+		return ACCESS_LEVEL_ADMIN;
+	if(!str_comp_nocase(pAccessLevel, "1"))
+		return ACCESS_LEVEL_MOD;
+	if(!str_comp_nocase(pAccessLevel, "2"))
+		return ACCESS_LEVEL_HELPER;
+	if(!str_comp_nocase(pAccessLevel, "3"))
+		return ACCESS_LEVEL_USER;
+
+	// new string access levels
```

Comment:
```suggestion
	// string access levels
```
"new" gets old pretty fast. ^^